src/AutoMapper/Configuration/IProfileConfiguration.cs

```diff
@@ -39,6 +39,12 @@ public interface IProfileConfiguration
         /// </summary>
         Func<FieldInfo, bool> ShouldMapField { get; }
 
+        /// <summary>
+        /// Specify which constructors should be mapped.
+        /// By default all non-static constructors are mapped.
+        /// </summary>
+        Func<ConstructorInfo, bool> ShouldMapCtor { get; }
```

Comment:
I think ShouldUseConstructor is a better name.

Reply:
Agreed, I used Ctor because other places in AM use Ctor in place of Constructor.

Reply:
Yes, an oversight :)